contracts/test/gmp/DestinationChainSwapExecutable.sol

```diff
@@ -22,13 +24,20 @@ contract DestinationChainSwapExecutable is AxelarExecutable {
     ) internal override {
         (string memory tokenSymbolB, string memory recipient) = abi.decode(payload, (string, string));
 
-        address tokenA = gateway.tokenAddresses(tokenSymbolA);
-        address tokenB = gateway.tokenAddresses(tokenSymbolB);
+        address tokenA = IAxelarGMPGatewayWithToken(address(gatewayAddress)).tokenAddresses(tokenSymbolA);
+        address tokenB = IAxelarGMPGatewayWithToken(address(gatewayAddress)).tokenAddresses(tokenSymbolB);
 
         IERC20(tokenA).approve(address(swapper), amount);
         uint256 convertedAmount = swapper.swap(tokenA, tokenB, amount, address(this));
 
-        IERC20(tokenB).approve(address(gateway), convertedAmount);
-        gateway.sendToken(sourceChain, recipient, tokenSymbolB, convertedAmount);
+        IERC20(tokenB).approve(address(gatewayAddress), convertedAmount);
```

Comment:
```suggestion
        IERC20(tokenB).approve(address(gateway()), convertedAmount);
```
instead of referring to `gatewayAddress` directly

Reply:
updated.